include/substrait/common/Decimal.h

```diff
@@ -0,0 +1,38 @@
+/* SPDX-License-Identifier: Apache-2.0 */
+
+#pragma once
+
+#include <string>
+
+namespace substrait::proto {
+  class Expression_Literal_Decimal;
+}
+
+namespace io::substrait::common {
+
+class Decimal {
```

Comment:
I think this class should be placed under the expression directory, which  represent the c++ classes corresponding to the substrait expressions?